README.md

```diff
@@ -1 +1,57 @@
-# React-Redux Space Travelers-Hub
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# SPACE TRAVELERS' HUB
+
+> This educational project is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+
+## Built With
+
+- ReactJS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- `git@github.com:myaserkhan/React-Redux-Space-Travelers-Hub.git`
+- `cd React-Redux-Space-Travelers-Hub`
+- `npm install`
+- `npm start`
+
+## Prerequisites
+
+- Node.js
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@myaserkhan](https://github.com/githubhandle)
+- Twitter: [@khanyaser007](https://twitter.com/twitterhandle)
+- LinkedIn: [myaserkhan](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@johnkioko](https://github.com/githubhandle)
+- Twitter: [@johnmutisya](https://twitter.com/twitterhandle)
+- LinkedIn: [John Mutisya](https://linkedin.com/in/linkedinhandle)
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly add the correct GitHub handles to your readme file so as to make it more professional.